util/missingcrypto.txt

```diff
@@ -1,1423 +1,1423 @@
-X509_add1_trust_object
-X509_alias_get0
-X509_alias_set1
-X509_aux_print
-X509_certificate_type
-X509_chain_check_suiteb
-X509_check_akid
-X509_check_purpose
-X509_check_trust
-X509_email_free
-X509_find_by_issuer_and_serial
-X509_find_by_subject
-X509_get0_pubkey_bitstr
-X509_get0_reject_objects
-X509_get0_trust_objects
-X509_get1_email
-X509_get1_ocsp
-X509_get_default_cert_area
-X509_get_default_cert_dir
-X509_get_default_cert_dir_env
-X509_get_default_cert_file
-X509_get_default_cert_file_env
-X509_get_default_private_dir
-X509_get_pubkey_parameters
-X509_get_signature_type
-X509_gmtime_adj
-X509_http_nbio
-X509_issuer_and_serial_hash
-X509_issuer_name_hash
-X509_issuer_name_hash_old
-X509_it
-X509_keyid_get0
-X509_keyid_set1
-X509_ocspid_print
-X509_policy_check
-X509_policy_level_get0_node
-X509_policy_level_node_count
-X509_policy_node_get0_parent
-X509_policy_node_get0_policy
-X509_policy_node_get0_qualifiers
-X509_policy_tree_free
-X509_policy_tree_get0_level
-X509_policy_tree_get0_policies
-X509_policy_tree_get0_user_policies
-X509_policy_tree_level_count
-X509_print
-X509_print_ex
-X509_print_ex_fp
-X509_print_fp
-X509_reject_clear
-X509_signature_dump
-X509_signature_print
-X509_subject_name_hash
-X509_subject_name_hash_old
-X509_supported_extension
-X509_to_X509_REQ
-X509_trust_clear
-X509_trusted
-X509at_add1_attr
-X509at_add1_attr_by_NID
-X509at_add1_attr_by_OBJ
-X509at_add1_attr_by_txt
-X509at_delete_attr
-X509at_get0_data_by_OBJ
-X509at_get_attr
-X509at_get_attr_by_NID
-X509at_get_attr_by_OBJ
-X509at_get_attr_count
-X509v3_addr_add_inherit
-X509v3_addr_add_prefix
-X509v3_addr_add_range
-X509v3_addr_canonize
-X509v3_addr_get_afi
-X509v3_addr_get_range
-X509v3_addr_inherits
-X509v3_addr_is_canonical
-X509v3_addr_subset
-X509v3_addr_validate_path
-X509v3_addr_validate_resource_set
-X509v3_asid_add_id_or_range
-X509v3_asid_add_inherit
-X509v3_asid_canonize
-X509v3_asid_inherits
-X509v3_asid_is_canonical
-X509v3_asid_subset
-X509v3_asid_validate_path
-X509v3_asid_validate_resource_set
-ZINT32_it
-ZINT64_it
-ZLONG_it
-ZUINT32_it
-ZUINT64_it
-a2d_ASN1_OBJECT
-a2i_ASN1_ENUMERATED
-a2i_ASN1_INTEGER
-a2i_ASN1_STRING
-a2i_GENERAL_NAME
-a2i_IPADDRESS
-a2i_IPADDRESS_NC
-b2i_PVK_bio
-b2i_PrivateKey
-b2i_PrivateKey_bio
-b2i_PublicKey
-b2i_PublicKey_bio
-conf_ssl_get
-conf_ssl_get_cmd
-conf_ssl_name_find
-d2i_X509_bio
-d2i_X509_fp
-err_free_strings_int
-i2a_ACCESS_DESCRIPTION
-i2a_ASN1_ENUMERATED
-i2a_ASN1_INTEGER
-i2a_ASN1_OBJECT
-i2a_ASN1_STRING
-i2b_PVK_bio
-i2b_PrivateKey_bio
-i2b_PublicKey_bio
-i2d_PrivateKey_bio
-i2d_PrivateKey_fp
-i2d_X509_bio
-i2d_X509_fp
-i2o_ECPublicKey
-i2s_ASN1_ENUMERATED
-i2s_ASN1_ENUMERATED_TABLE
-i2s_ASN1_IA5STRING
-i2s_ASN1_INTEGER
-i2s_ASN1_OCTET_STRING
-i2v_ASN1_BIT_STRING
-i2v_GENERAL_NAME
-i2v_GENERAL_NAMES
-o2i_ECPublicKey
-s2i_ASN1_IA5STRING
-s2i_ASN1_INTEGER
-s2i_ASN1_OCTET_STRING
-v2i_ASN1_BIT_STRING
-v2i_GENERAL_NAME
-v2i_GENERAL_NAMES
-v2i_GENERAL_NAME_ex
+ACCESS_DESCRIPTION_it(3)
```

Comment:
Since the items in these missingXXXXX files all claim to be *functions* is it really necessary to add the "(3)" after everything?  Can't find-doc-nits append the "(3)" if it needs it internally?  Are you thinking that at some point we'll want "(7)" or such?

Reply:
I'm thinking of that possibility, yeah, even if it currently seems remote.

Reply:
So maybe, if/when you want things like "(7)" then make the change at that time?

Reply:
And why not now?  I've already made the change, are you actually asking me to undo it?

Reply:
I don't get a vote, but I think it's a needless change.

Reply:
I disagree.  It makes things clearer.